### PR TITLE
Added Shortest_Job_First Algorithm

### PR DESCRIPTION
**Pull request by pharshil1902**

### CODE REVIEW

File: scheduling/Shortest_Job_First Algorithm.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        total_waiting_time = total_waiting_time + waiting_time[i] 
+        total_turn_around_time = total_turn_around_time + turn_around_time[i] 
+
+
+    print("\nAverage waiting time = %.5f "%(total_waiting_time /no_of_processes) ) 
+    print("Average turn around time = ", total_turn_around_time / no_of_processes) 
+    
+
+print("Enter How Many Process You want To Enter")
+no_of_processes=int(input())
```
Comment (cclauss) [inline scheduling/Shortest_Job_First Algorithm.py:118]: This line will fail our automated testing as discussed in CONTRIBUTING.md.

File: scheduling/Shortest_Job_First Algorithm.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+for i in range(n):
+    print("Enter The Brust time and Arrival time for Process:--"+str(i+1))
+    burst_time[i],arrival_time[i]=map(int,input().split())
+    
+wt=findWaitingTime(processes, n, burst_time, arrival_time)
+tat= findTurnAroundTime(processes, n, burst_time, wt)
+compl_time=findavgTime(processes, n, burst_time,arrival_time,wt,tat)
+
+FCFS=pd.DataFrame(list(zip(processes,burst_time,arrival_time,wt,tat,compl_time)),columns=['Processes','Burst Time','Arrival Time','Waiting Time','Turn Around Time','complition Time'])
```
Comment (mrmaxguns) [inline scheduling/Shortest_Job_First Algorithm.py:201]: It seems like this line is too long

**Comment by cclauss** [comment]
As discussed in CONTRIBUTING.md, running the code thru `psf/black` should replace tab indentation with spaces and wrap most lines to 88 characters.

**Comment by pharshil1902** [comment]
> As discussed in CONTRIBUTING.md, running the code thru `psf/black` should replace tab indentation with spaces and wrap most lines to 88 characters.

File: scheduling/Shortest_Job_First Algorithm.py (hunk 2 above)
Comment (pharshil1902) [reply scheduling/Shortest_Job_First Algorithm.py:201]: Thanks a lot. I'll work on it

File: scheduling/Shortest_Job_First Algorithm.py (hunk 1 above)
Comment (pharshil1902) [reply scheduling/Shortest_Job_First Algorithm.py:118]: Thanks a lot. I'm new to these flake Rules

File: scheduling/Shortest_Job_First Algorithm.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    total_turn_around_time = 0
+    for i in range(no_of_processes): 
+        total_waiting_time = total_waiting_time + waiting_time[i] 
+        total_turn_around_time = total_turn_around_time + turn_around_time[i] 
+    print("\nAverage waiting time = %.5f "%(
+    total_waiting_time /no_of_processes) ) 
+    print("Average turn around time = ", 
+    total_turn_around_time / no_of_processes) 
+    
+
```
Comment (cclauss) [inline scheduling/Shortest_Job_First Algorithm.py:112]: Please indent all the remaining lines under:
`if __name__ == "__main__":`

**Comment by pharshil1902** [comment]
Thanks a Lot for your help.

**Pull request closed without merging**